pkg/bpf/map_linux_test.go

```diff
@@ -1025,31 +1025,38 @@ func TestBatchIteratorTypes(t *testing.T) {
 	iter := NewBatchIterator[TestKey, TestValue](m)
 	iter.IterateAll(context.TODO())
 	assert.Error(t, iter.Err())
+	assert.NotNil(t, iter)
 }
 
 func TestBatchIterator(t *testing.T) {
 	testutils.PrivilegedTest(t)
 
-	runTest := func(mapType ebpf.MapType, size, mapSize int, t *testing.T, opts ...BatchIteratorOpt[TestKey, TestValue, *TestKey, *TestValue]) {
+	runTest := func(mapType ebpf.MapType, size, mapSize int, t *testing.T, opts ...BatchIteratorOpt[TestLPMKey, TestValue, *TestLPMKey, *TestValue]) {
+		makeKey := func(i int) MapKey {
+			return &TestLPMKey{
```

Comment:
Nit: from a readability perspective it feels a bit strange to use TestLPMKey for all map types.